src/js/components/DataTable/StyledDataTable.js

```diff
@@ -103,12 +108,18 @@ const StyledDataTableBody = styled(TableBody)`
     overflow: auto;
   `}
 
-  &:focus {
```

Comment:
Why did we not keep focus at the table body?

Reply:
This is back in place now but utilizing the same approach as List.js where it'll look like:

```js
&:focus {
  ${props => props.tabIndex >= 0 && focusStyle ...}
}
```